test/karma-system-config.js

```diff
@@ -96,6 +96,8 @@ System.config({
     '@angular/cdk-experimental/scrolling': 'dist/packages/cdk-experimental/scrolling/index.js',
 
     '@angular/material/autocomplete': 'dist/packages/material/autocomplete/index.js',
+    '@angular/material/autocomplete/testing': 'dist/packages/material/autocomplete/testing/index.js',
+    '@angular/material/autocomplete/testing/shared.spec': 'dist/packages/material/autocomplete/testing/shared.spec.js',
```

Comment:
Ugh, not really nice that we have to add this for each harness we move, but yeah, I see why we need it.